packages/loopback/lib/context/binding.ts

```diff
@@ -5,21 +5,15 @@
 
 export class Binding {
   public value: any;
-  private _isDynamic: boolean = false;
+  public getValue: () => any = () => { throw new Error(`No value was configured for binding {this._key}.`); };
```

Comment:
I see now that it actually _does_ have a different meaning in **this particular** scenario. :-(